ccViewer/ccViewerApplication.h

```diff
@@ -36,6 +36,8 @@ class ccViewerApplication : public ccApplicationBase
 	bool event( QEvent *inEvent ) override;
 	
 private:
+    // ptr to QMainWindow->ccViewer
```

Comment:
Format is //! and once again I wouldn't remind here that ccViewer derives from QMainWindow (I'm not sure to understand the benefit?). 
--> '//! Associated ccViewer instance'?